### ISSUE TITLE
I want to use failover mode and load balancing mode of MariaDB connector / j.

### ISSUE THREAD

**Issue by TadashiKanda** [mybatis/guice#373]
I'm using AWS RDS (aurora).
I want to use failover mode and load balancing mode of MariaDB connector / j.

However, using the mariadb connector / j does not work as expected.

Looking at the source below, I realized that I needed to make some changes.

★Source
 guice/src/main/java/org/mybatis/guice/datasource/helper/JdbcHelper.java

>★Change before
>　83  MariaDB("jdbc:mysql://${JDBC.host|localhost}:${JDBC.port|3306}/${JDBC.schema}", "org.mariadb.jdbc.Driver"),
>★Change after
>　83  MariaDB("jdbc:mysql:aurora://${JDBC.host|localhost}:${JDBC.port|3306}/${JDBC.schema}", "org.mariadb.jdbc.Driver"),

https://mariadb.com/kb/en/about-mariadb-connector-j/#failover-and-load-balancing-modes

Because there are multiple parameters, I hope I can set it later.

Is there a way?
Thank you.

**Comment by TadashiKanda** [comment]
If I use a binder, can I use the specified method?

The functions will conflict.

__JdbcHelper.java__
```Java  @Override
  public void configure(Binder binder) {
    binder.bindConstant().annotatedWith(named(JDBC_DRIVER)).to(driverClass);
    binder.bind(Key.get(String.class, named(JDBC_URL))).toProvider(guicify(new JdbcUrlAntFormatter(urlTemplate)));
  }`

**Comment by christianpoitras** [comment]
I would not use JdbcHelper at all since it should only be used if your URL matches the one provided by the class.
If you look at the examples on the links I provided, the JdbcHelper class is not used.

**Comment by TadashiKanda** [comment]
I understand. The proposed content requires hard coding for each connection destination, which is not realistic.

**Comment by TadashiKanda** [comment]
We are confident that adding this will bring many benefits to anyone who uses or plans to use AWS Aurora.
Thank you in advance for your consideration.

**Comment by christianpoitras** [comment]
I see your point. I will probably add these values to the `JdbcHelper` class.

**Comment by TadashiKanda** [comment]
Thank you! Are you sure you want to send me a pull request?

**Comment by christianpoitras** [comment]
You can submit a pull request if you want. That would be easier for me. :)

**Comment by TadashiKanda** [comment]
You have sent a pull request. Please check it.

**Comment by christianpoitras** [comment]
Fixed in revision e2a9906863edf1b95ca2c702e802b52300bf19fb.
